### PR TITLE
Add JobId class into applicant

### PR DESCRIPTION
**Pull request by seanleong339**
Add JobId class, change editcommand and addapplicant method to accomodate

### CODE REVIEW

**Review by e0543517 — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/applicant/AddApplicantParser.java [hunk 1]
```diff
@@ -43,10 +42,10 @@ public AddApplicant parse(String args) throws ParseException {
...
                         PREFIX_ADDRESS, PREFIX_TAG, PREFIX_DATEAPPLIED, PREFIX_DATEINTERVIEW, PREFIX_NRIC,
-                        PREFIX_JOB, PREFIX_QUALIFICATION
+                        PREFIX_QUALIFICATION
                 );
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_JOB,
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL,
             PREFIX_NRIC, PREFIX_DATEAPPLIED)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddApplicant.MESSAGE_USAGE));
```
Comment (Justinnje) [inline src/main/java/seedu/address/logic/parser/applicant/AddApplicantParser.java:51]: Not sure if u changed the Message_usage message in addapplicant class. Should remove the jobid requirement in the string

**Review by Justinnje — COMMENTED** [review]
Just 1 small change in addapplicant class

File: src/main/java/seedu/address/logic/parser/applicant/AddApplicantParser.java (hunk 1 above)
Comment (seanleong339) [reply src/main/java/seedu/address/logic/parser/applicant/AddApplicantParser.java:51]: Ok changed

**Review by Justinnje — APPROVED** [review]
LGTM

**Pull request merged**